### PR TITLE
Soap is no longer comdom scented. Collectable captain hat doesn't look like a comdom hat.

### PR DESCRIPTION
**Pull request by Fordoxia**
## What Does This PR Do
Removes comdoms. Also the deluxe soap is not made by Waffle Co. anymore because the lore team said so.

## Why It's Good For The Game
Comdom scent is weird. Waffle co are criminals and have been for a while, so it's weird that NT would buy soap from them.

## Testing
Couldn't smell comdom.

## Changelog
:cl:
tweak: Removed comdom references from deluxe soap and collectable captain hat.
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/soap.dm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/obj/item/soap
+code/modules/clothing/head/collectable.dm/obj/item/soap
```
Comment (DGamerL) [inline code/game/objects/items/weapons/soap.dm:1]:
```suggestion
/obj/item/soap
```
I think this would be a more sane way to write this down

**Review by DGamerL — CHANGES_REQUESTED** [review]

File: code/game/objects/items/weapons/soap.dm (hunk 1 above)
Comment (Fordoxia) [reply code/game/objects/items/weapons/soap.dm:1]: I don't know why this is even here I hate it.

**Review by Henri215 — APPROVED** [review]
Looks comdom to me

**Review by DGamerL — APPROVED** [review]
I 100% agree on the soap description, but the hat _does_ make you look like a comdom

**Comment by Fordoxia** [comment]
> I 100% agree on the soap description, but the hat _does_ make you look like a comdom

Pope did ask that it doesn't call you a comdom. Even if it *is* true.

**Comment by PopeDaveThe3th** [comment]
> > I 100% agree on the soap description, but the hat _does_ make you look like a comdom
> 
> 
> 
> Pope did ask that it doesn't call you a comdom. Even if it *is* true.

literally 1984

**Comment by matttheficus** [comment]
My SOUL!!!!!!

**Review by lewcc — APPROVED** [review]

**Pull request merged**